sklearn/metrics/classification.py

```diff
@@ -1544,7 +1544,8 @@ def hamming_loss(y_true, y_pred, classes=None, sample_weight=None):
         raise ValueError("{0} is not supported".format(y_type))
 
 
-def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
+def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
+             labels=None):
     """Log loss, aka logistic loss or cross-entropy loss.
 
     This is the loss function used in (multinomial) logistic regression
```

Comment:
I think the docstring should document the assumptions. I'm not sure we do this in other places but it would be good practice. If labels is None and y_pred is 1-d, what happens? This is sort of no-obvious. Also, the labels in y_pred are assumed to be ordered alphabetically, as done by ``LabelBinarizer``, right?